via/services/url_details.py

```diff
@@ -21,19 +28,19 @@ def get_url_details(self, url, headers=None):
         :return: 2-tuple of (mime type, status code)
 
         :raise BadURL: When the URL is malformed
+        :raise checkmatelib.BadURL: if the URL is blocked by Checkmate
         :raise UpstreamServiceError: If we server gives us errors
         :raise UnhandledException: For all other request based errors
         """
-        if headers is None:
-            headers = OrderedDict()
```

Comment:
Merged this `if` into the `headers = ` line below

Reply:
:+1: Could go one further and inline it

Reply:
:+1: https://github.com/hypothesis/via/pull/1009/commits/79fd557a9c0adbf895f7cf11299276ff3235367b